Packages/com.unity.multiplayer.samples.coop/Utilities/EditorChildSceneLoader.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+/// <summary>
+/// Should be placed inside root scene and setup with child scenes to load. ChildSceneLoader will take care of the rest
+/// </summary>
+public class EditorChildSceneLoader : MonoBehaviour
```

Comment:
I'd say that we want this for runtime too, no?

Reply:
In runtime it would execute in awake and load all the other scenes according to whatever scene loading rules we decide to define (right now it's just "load everything from the list")

Reply:
This would mess with NGO's scene loading.
Another thing to note is runtime scene loading flows could get complex, especially with my current DGS work. With DGS, the scene loaded will change whether you're a client or a server.
With boss room, our additive scenes are loaded only once we reach a certain trigger in the map, they are not all loaded at once.
This here is really just to for authoring setup. "This is want I want to edit when I'm editing this group of scenes".